### PR TITLE
implement feature add student

### PR DESCRIPTION
**Pull request by lyratadev**
implement feature add student #1 
STORY 
Admin can add new students to the database by pressing a button and inputting their email as well as name. 

We utilize react components and a route to enable functionality where the admin can press said button to add a student. Upon success the toast component will inform the user; if failure occurs the server will return an html status code for the failure.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -20,6 +22,8 @@ function App() {
        <Switch>
         <Route exact path='/' component={Semester} />
         <Route path='/schedule' component={SchedList} />
+        <Route path='/addstudent' component={AddStudent} />
+
```
Comment (andrewpaulino) [inline src/App.js:26]: NIT: I'd remove this extra line.

Comment (andrewpaulino) [inline src/App.js:26]: NIT: I'd remove this extra line.

File: src/components/AddStudent.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+              </Button>
+              <ToastContainer autoClose={1500} style={{ top: '50%', left: '50%', transform: 'translate(-50%, -50%)' }}/> 
+          </div>
+        </div>
+      ); 
+    }
+}
+
+
+export default AddStudent;
```
Comment (andrewpaulino) [inline src/components/AddStudent.js]: Add new line here

**Review by andrewpaulino — APPROVED** [review]
Looks good! Added a few nit pick commments.

File: src/components/AddStudent.js (hunk 2 above)
Comment (lyratadev) [reply src/components/AddStudent.js]: noted.

**Pull request merged**